portfolio/src/main/webapp/script.js

```diff
@@ -80,25 +66,16 @@ yogaReveal.onclick = function() {
 }
 
 
-function getName() {
-    console.log('Fetching Name');
-
-    // request html from servlet data
-    const responsePromise = fetch('/data');
+// To parse JSON into a usable object for javascript
+function getComments() {
 
-    responsePromise.then(handleResponse);
+    // fetch data from server, parse as json file, then reference coms as an object
+    fetch('/data').then(response => response.json()).then((comments) => {
+        const commentList = document.getElementById("comment-container");
+        console.log("fetching comments");
+        comments.forEach((comment) => {
```

Comment:
I think you deleted createListElement with you last commit.  After you add that back this is good to submit.

Reply:
Yes, I noted this when I tried to run it, its added back in the next push.